pkg/api/v1/types.go

```diff
@@ -100,11 +100,14 @@ func init() {
 	})
 }
 
+// VM is *the* VM Definition. It represents a virtual machine in the runtime environment of kubernetes.
 type VM struct {
 	metav1.TypeMeta `json:",inline"`
 	ObjectMeta      metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec            VMSpec            `json:"spec,omitempty" valid:"required"`
-	Status          VMStatus          `json:"status"`
+	// Spec contains the low level VM specification that can be understood by underlying libraries.
```

Comment:
Trimmed the doc to get rid of the "distinction", more information can be found under the struct itself.